parla/task_runtime.py

```diff
@@ -379,6 +380,8 @@ def run(self):
                 # the scheduler.
                 for d in self.req.devices:
                     ctx.scheduler._available_resources.deallocate_resources(d, self.req.resources)
+                    # TODO: This is part of the easy launcher hack, fix it
+                    ctx.scheduler._available_resources._occupancy_dict[d] = False
```

Comment:
I currently keep a (hacky) dict in the scheduler listing which resources are free or not. Later I will update this to properly use the ResourcePool.